### PR TITLE
docs: updated omnipool

### PR DESCRIPTION
**Pull request by enthusiastmartin**
Update code docs and readme to be up to date with implementation.

### CODE REVIEW

File: pallets/omnipool/src/lib.rs [hunk 1]
```diff
@@ -349,23 +367,19 @@ pub mod pallet {
 		AssetAlreadyAdded,
 		/// Asset is not in omnipool
 		AssetNotFound,
-		/// No stable asset in the pool
-		NoStableAssetInPool,
```
Comment (Roznovjak) [inline pallets/omnipool/src/lib.rs:353]: This is a breaking change

File: pallets/omnipool/src/lib.rs [hunk 2]
```diff
@@ -1420,6 +1425,17 @@ pub mod pallet {
...
+		/// Removes protocol liquidity.
+		///
+		/// Protocol liquidity is liquidity from sacrificed positions. In order to remove protocol liquidity,
+		/// we need the know the price of the position at the time of sacrifice. Hence this specific call.
+		///
+		/// Only `AuthorityOrigin` can perform this call.
+		///
+		/// Note that sacrifice position will be deprecated in future. There is no longer a need for that.
+		///
+		/// IT works the same way as remove liquidity call, but position is temporary reconstructed.
```
Comment (Roznovjak) [inline pallets/omnipool/src/lib.rs:1437]:
```suggestion
		/// It works the same way as remove liquidity call, but position is temporary reconstructed.
```

File: pallets/omnipool/src/lib.rs [hunk 3]
```diff
@@ -21,50 +21,66 @@
...
+//! * **Imbalance:** Tracking of hub asset imbalance.
 //!
 //! ## Assumptions
 //!
 //! Below are assumptions that must be held when using this pallet.
 //!
-//! * First two asset in pool must be Stable Asset and Native Asset. This must be achieved by calling
-//!   `initialize_pool` dispatchable.
-//! * Stable asset balance and native asset balance must be transferred to omnipool account manually.
-//! * All tokens added to the pool must be first registered in Asset Registry.
```
Comment (Roznovjak) [inline pallets/omnipool/src/lib.rs:52]: Do we really want to remove it?

File: pallets/omnipool/src/lib.rs (hunk 1 above)
Comment (mrq1911) [reply pallets/omnipool/src/lib.rs:353]: that is just cleanup, it wasn't used anymore

**Review by mrq1911 — APPROVED** [review]

File: pallets/omnipool/src/lib.rs (hunk 1 above)
Comment (Roznovjak) [reply pallets/omnipool/src/lib.rs:353]: Yes, but it changes the index numbers of errors. Maybe it's not a problem.

File: pallets/omnipool/src/lib.rs (hunk 3 above)
Comment (enthusiastmartin) [reply pallets/omnipool/src/lib.rs:52]: it is no longer valid.

**Pull request merged**